typeDefs.js

```diff
@@ -386,6 +394,7 @@ const typeDefs = gql`
 
     "User Mutation"
     registerUser(user: userRegisterInput!, language: Int!): User
+    registerAdmin(user: SpecialUserRegisterInput!): UserResult
```

Comment:
Please, make `LogicalResult` not-nullable

Reply:
fixed